fog/distribution/src/main.rs

```diff
@@ -238,6 +238,39 @@ fn main() {
 
     let mut running_threads: usize = 0;
 
+    // Seed each fog account with a TxOut. This ensures that integration tests
+    // that check to make sure each fog account has a non-zero balance do not
+    // fail.
+    let mut seed_fog_resolver = build_fog_resolver(&fog_uri, &env, &logger);
+    let conns = get_conns(&config, &logger);
+
+    log::info!(logger, "Seeding Fog Accounts with initial TxOuts.");
+    for (i, fog_account) in fog_accounts.iter().enumerate() {
+        if let Some(new_fog_resolver) = build_and_submit_transaction(
+            i,
+            // For this seed phase, only use one TxOut for each transaction.
+            vec![spendable_txouts[i].clone()],
+            fog_account,
+            &config,
+            &ledger_db,
+            &seed_fog_resolver,
+            &logger,
+            &conns,
+            &env,
+            &fog_uri,
+        ) {
+            seed_fog_resolver = new_fog_resolver;
+        }
+    }
+
+    // Don't use spendable_txouts that were used in the seed step.
+    spendable_txouts = spendable_txouts[fog_accounts.len()..].to_vec();
```

Comment:
Perhaps there is a slicker way to do not use the SpendableTxOuts that were used in the pre-slam debit phase?

Reply:
It might be cleaner to split this into a seed txouts and slam txouts before you use them for seeding instead of after.

Reply:
I prefer the way it's written to that option because in that option we'd have to introduce logic that's like "For the first x spendable tx outs, where x == fog acocunts length, add to this one vec, and then for every spendable tx out after this x add to another vec." It seems cleaner to me to work off one vec of spendable txouts.

But this seems to just be based on a preference of mine and not grounded in any hard evidence etc.

Reply:
I'm fine either way.